modules/cms-kit/src/Volo.CmsKit.Public.Web/Menus/CmsKitPublicMenuContributor.cs

```diff
@@ -22,7 +24,8 @@ public async Task ConfigureMenuAsync(MenuConfigurationContext context)
 
     private async Task ConfigureMainMenuAsync(MenuConfigurationContext context)
     {
-        if (GlobalFeatureManager.Instance.IsEnabled<MenuFeature>())
+        var featureChecker = context.ServiceProvider.GetRequiredService<IFeatureChecker>();
+        if (GlobalFeatureManager.Instance.IsEnabled<MenuFeature>() && featureChecker.IsEnabledAsync(CmsKitFeatures.MenuEnable))
```

Comment:
I think there is a missing `await` usage
```suggestion
        if (GlobalFeatureManager.Instance.IsEnabled<MenuFeature>() && await featureChecker.IsEnabledAsync(CmsKitFeatures.MenuEnable))
```

Reply:
you are right